src/components/core/Markdown/MarkdownShortcuts.tsx

```diff
@@ -1,30 +1,34 @@
+import { captureException } from '@sentry/nextjs';
 import React, { useEffect, useState } from 'react';
 import styled from 'styled-components';
 
 import Bold from './Bold';
 import Link from './Link';
 import List from './List';
 
-export default function MarkdownShortcuts({
-  textAreaRef,
-  onChange, // The onChange from the TextArea component. This ensures that we track the same event handlers (e.g. a collector's note change) via the addition of just markdown
-}: {
+// Use the TextArea element's native setter and dispatch an input event to trigger the onChange callback
+export function setValueAndTriggerOnChange(textArea: HTMLTextAreaElement, newValue: string) {
+  var nativeTextAreaValueSetter = Object.getOwnPropertyDescriptor(
+    HTMLTextAreaElement.prototype,
+    'value'
+  )?.set;
+
+  if (!nativeTextAreaValueSetter) {
```

Comment:
added this condition to satisfy typescript - nativeTextAreaValueSetter could be undefined.
added to sentry because I'm curious if this will happen in the wild

Reply:
what is this sorcery?!?!?